addons/l10n_eu_service/models/res_company.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from .eu_tax_map import EU_TAX_MAP
+
+
+class Company(models.Model):
+    _inherit = 'res.company'
+
+    def _get_repartition_line_copy_values(self, original_rep_lines):
+        return [(0, 0, {
+            'factor_percent': line.factor_percent,
+            'repartition_type': line.repartition_type,
+            'account_id': line.repartition_type == 'tax' and line.account_id.id or None,
+            'company_id': line.company_id.id,
+            'sequence': line.sequence,
+        }) for line in original_rep_lines]
+
+    def map_eu_taxes(self):
+        eu_countries = self.env.ref('base.europe').country_ids
+        tax_group_id = self.env.ref('l10n_eu_service.tax_group_oss').id
+        for company in self:
+            tax_ids = self.env['account.tax'].search([('type_tax_use', '=', 'sale'),
```

Comment:
When on fields, the convention is indeed that ..._id and ..._ids denote recordsets. For variable, rather use _id and _ids are used when you actually are manipulating ids. So here, tax_group_id makes sense, but tax_ids should just be taxes, or domestic_taxes (longer, but clearer).

Reply:
taxes work